File: Ej 2 (Anagrama)/Python/Anagrama.py
## DeathbatO

def Anagrama(palabra1, palabra2):
    if (palabra1 == palabra2) or (len(palabra2) != len(palabra1)):
        return False
    else:
        count = 0
        for i in range(len(palabra1)):
            for j in range(len(palabra2)):
                if palabra1[i] == palabra2[j]:
                    count += 1
                    break
        if count >= len(palabra1):
            return True

print("Vamos a verificar si dos palabras son un anagrama")
palabra1 = input("Ingrese la primera palabra: ")
palabra2 = input("Ingrese la segunda palabra: ")

print(Anagrama(palabra1, palabra2))

File: Ej 2 (Anagrama)/Kotlin/Anagrama.kt
//DeathbatO

fun sonAnagramas(palabra1: String, palabra2: String): Boolean {
    if (palabra1 == palabra2 || palabra1.length != palabra2.length) {
        return false
    } else {
        var count = 0
        for (i in palabra1.indices) {
            for (j in palabra2.indices) {
                if (palabra1[i] == palabra2[j]) {
                    count++
                    break
                }
            }
        }
        return count >= palabra1.length
    }
}

fun main() {
    println("Vamos a verificar si dos palabras son un anagrama")
    print("Ingrese la primera palabra: ")
    val palabra1 = readLine() ?: ""
    print("Ingrese la segunda palabra: ")
    val palabra2 = readLine() ?: ""

    println(sonAnagramas(palabra1, palabra2))
}